### PR TITLE
Ledger support for unbond and withdraw

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**
Support for ledged users to unbond and withdraw their tokens. @Kahonnohak please update text.
image

Warning for Ledger native users
image

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -886,6 +879,13 @@ export default {
     unbondFrom: 'Unbond from {name}',
     startUnbonding: 'Start unbonding',
     unbondingEra: 'Unbonding takes {unbondingPeriod} eras before you can withdraw',
+    migrationSupport: {
+      actionRequired: 'Action Required',
+      yourTokensAreLocked:
+        'Your tokens are locked in dAppStaking V2. Please unlock and withdraw your tokens. Since Ledger is not supported yet for dApp staking v3 move your funds to a soft wallet to be able to participate in dApp staking.',
+      migrateNow: 'Migrate Now',
+    },
+    ledgerNotSupported: 'Ledger native accounts is not supported yet for dApp staking v3 yet.',
```
Comment (impelcrypto) [inline src/i18n/en-US/index.ts:888]:
```suggestion
    ledgerNotSupported: 'Ledger native accounts are not supported for dApp staking V3 yet.',
```

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Comment by Kahonnohak** [comment]
Hi @bobo-k2 for the UI message

- Lets mention EVM
- Add link to docs : https://docs.astar.network/docs/learn/dapp-staking/dapp-staking-faq/#q-i-am-a-leger-astar-native-app-user-what-do-i-need-to-do

How about message below (took same sentence from the docs) :
Your tokens are locked in dAppStaking V2. Please unbond and withdraw your tokens. dApp Staking V3 is temporally unavailable for those Ledger Astar Native App users, please move your funds to a soft wallet or a Ledger EVM account to be able to participate in dApp staking. (More information)

**Comment by bobo-k2** [comment]
> Hi @bobo-k2 for the UI message

Updated in https://github.com/AstarNetwork/astar-apps/pull/1158/commits/3b5d7db0896d37be8884281feb639eeac120b2ff

File: src/hooks/useAccount.ts [hunk 2]
```diff
@@ -139,7 +139,12 @@ export const useAccount = () => {
 
   const showAccountUnificationModal = (): void => {
     const eventAggregator = container.get<IEventAggregator>(Symbols.EventAggregator);
-    eventAggregator.publish(new UnifyAccountMessage());
+    eventAggregator.publish(
+      new ExtrinsicStatusMessage({
+        success: false,
+        message: 'Ledger is not supported on this network.',
```
Comment (impelcrypto) [inline src/hooks/useAccount.ts:145]: Let's use i18 here

File: src/staking-v3/components/my-staking/MigrationSupport.vue [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
       <div class="row--header">
-        {{ $t('dappStaking.migrationSupport.actionRequired') }}
+        {{ $t('stakingV3.migrationSupport.actionRequired') }}
       </div>
       <div class="row--body">
         <div class="text">
-          {{ $t('dappStaking.migrationSupport.yourTokensAreLocked') }}
+          {{ $t('stakingV3.migrationSupport.yourTokensAreLocked') }}
+          (<a
+            href="https://docs.astar.network/docs/learn/dapp-staking/dapp-staking-faq/#q-i-am-a-leger-astar-native-app-user-what-do-i-need-to-do"
```
Comment (impelcrypto) [inline src/staking-v3/components/my-staking/MigrationSupport.vue:11]: Let's import the url from `docsUrl` at `src/links.index`

**Review by impelcrypto — APPROVED** [review]
LGTM, please check my comments

**Review by ayumitk — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]
LGTM!

**Comment by bobo-k2** [comment]
> LGTM, please check my comments

Thanks, a lot. The first think you commented was bug. I am not sure why I made that change. Fixed in https://github.com/AstarNetwork/astar-apps/pull/1158/commits/d6711a6a6492123890ca03f47ea473cd6234b419

**Pull request merged**